pkg/resources/database_commons.go

```diff
@@ -89,7 +89,7 @@ func init() {
 			Name:         sdk.ObjectParameterLogLevel,
 			Type:         schema.TypeString,
 			Description:  fmt.Sprintf("Specifies the severity level of messages that should be ingested and made available in the active event table. Valid options are: %v. Messages at the specified level (and at more severe levels) are ingested. For more information, see [LOG_LEVEL](https://docs.snowflake.com/en/sql-reference/parameters.html#label-log-level).", sdk.AsStringList(sdk.AllLogLevels)),
-			ValidateDiag: StringInSlice(sdk.AsStringList(sdk.AllLogLevels), true),
+			ValidateDiag: sdkValidation(sdk.ToLogLevel),
 			DiffSuppress: func(k, oldValue, newValue string, d *schema.ResourceData) bool {
```

Comment:
Nit: if we change the validate, why don't adjust suppress with NormalizeAndCompare too?

Reply:
Done.